pandas/core/reshape.py

```diff
@@ -405,6 +406,9 @@ def _slow_pivot(index, columns, values):
 
 
 def unstack(obj, level, fill_value=None):
+    if fill_value:
```

Comment:
actually I would *always* pass this (we will make ``None`` an acceptable fill_value below)